js/samples/advanced-csharp/Pages/Options.cshtml

```diff
@@ -0,0 +1,141 @@
+        </section>
+        <div id='Cols'>
+            <section class='section' id='SampledisableGrammar'>
+                <div class='immersive-reader-button ir-button-small' onclick="handleLaunchImmersiveReader('disableGrammar')"></div>
+                <h3 id='titledisableGrammar'>Water (Parts of Speech, Syllables and Picture Dictionary are disabled - <span id='langdisableGrammar'>en</span>)</h3>
+                <p id='textdisableGrammar'>Fresh water sources also influence where people settle. People need water to drink. They also need it for washing. Throughout history, people have settled near fresh water. Living near a water source helps ensure that people have the water they need.</p>
+            </section>
+            <section class='section' id='SampledisableTranslation'>
+                <div class='immersive-reader-button ir-button-small' onclick="handleLaunchImmersiveReader('disableTranslation')"></div>
+                <h3 id='titledisableTranslation'>Mountain Ranges (Translation disabled - <span id='langdisableTranslation'>en</span>)</h3>
+                <p id='textdisableTranslation'>The physical features of a region are often rich in resources. Within a nation, mountain ranges become natural borders for settlement areas. In the U.S., major mountain ranges are the Sierra Nevada, the Rocky Mountains, and the Appalachians.</p>
+            </section>
+        </div>
+        <section class='section' id='Sampleparent'>
+            <div class='immersive-reader-button ir-button-small' onclick="handleLaunchImmersiveReader('parent')"></div>
+            <input type="checkbox" id="checkboxparent" name="Parent" value="true">
+            <label for="vehicleparent"> Launch in Parent </label><br>
+            <h3 id='titleparent'>Kaiju (Displaying in parent - <span id='langparent'>en</span>)</h3>
+            <p id='textparent'>Kaiju is a Japanese genre of films and television featuring giant monsters. The term kaiju can also refer to the giant monsters themselves, which are usually depicted attacking major cities and battling either with the military or other monsters. The kaiju genre is a subgenre of tokusatsu (special filming) entertainment.</p>
+            <div id="parentDiv" style="height: 400px; "></div>
+        </section>
+    </main>
+
+    <script type='text/javascript'>
+        async function handleLaunchImmersiveReader(sampleId) {
+            var langElement = $('#lang' + sampleId);
+            var lang = langElement.val() || langElement.text();
```

Comment:
Seems like only disableLangDetection sample needs this. Could just check for null and set 'en' by default. Remove lang spans from above where not needed

Reply:
done